Lib/random.py

```diff
@@ -719,7 +719,7 @@ def gammavariate(self, alpha, beta):
         if alpha > 1.0:
 
             # Uses R.C.H. Cheng, "The generation of Gamma
-            # variables with non-integral shape parameters",
```

Comment:
This looks like a reference that should not be changed

Reply:
Indeed,  thank you.